### PR TITLE
Missing legacy link for themes page

### PR DESCRIPTION
**Pull request by tomas862**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.6.x
| Description?  | Missing legacy links for theme controller
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | https://github.com/PrestaShop/PrestaShop/issues/13747
| How to test?  | Automated tests will handle all cases

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/14029)

### CODE REVIEW

**Review by matks — DISMISSED** [review]

File: src/PrestaShopBundle/Resources/config/routing/admin/improve/design/theme.yml [hunk 1]
```diff
@@ -12,27 +12,33 @@ admin_themes_upload_logos:
   defaults:
     _controller: PrestaShopBundle:Admin\Improve\Design\Theme:uploadLogos
     _legacy_controller: AdminThemes
+    _legacy_link: AdminThemes:submitOptionsconfiguration
```
Comment (matks) [inline src/PrestaShopBundle/Resources/config/routing/admin/improve/design/theme.yml:15]: Sure about this one ? `Theme:uploadLogos` and `AdminThemes:submitOptionsconfiguration` seem very different but I trust you 😄

**Comment by matks** [comment]
Travis says
```
1) Tests\Integration\PrestaShopBundle\Routing\Converter\LegacyUrlConverterTest::testConverterByParameters
Expected url /improve/design/themes/prestashop_theme/enable is different with generated one: /improve/design/themes/prestashop_theme/enable/
Failed asserting that false is true.
```
Is that the `/` at the end 🤔 ?

File: src/PrestaShopBundle/Resources/config/routing/admin/improve/design/theme.yml [hunk 2]
```diff
@@ -12,27 +12,33 @@ admin_themes_upload_logos:
...
 admin_themes_import:
   path: /import
   methods: [GET, POST]
   defaults:
     _controller: PrestaShopBundle:Admin\Improve\Design\Theme:import
     _legacy_controller: AdminThemes
+    _legacy_link: AdminThemes:importtheme
 
 admin_themes_enable:
   path: /{themeName}/enable/
```
Comment (PierreRambaud) [inline src/PrestaShopBundle/Resources/config/routing/admin/improve/design/theme.yml]:
```suggestion
  path: /{themeName}/enable
```

Comment (matks) [reply src/PrestaShopBundle/Resources/config/routing/admin/improve/design/theme.yml]: I'm not sure this changes something 🤔AFAIK Symfony does not care about trailing slashes, and the route below `path: /{themeName}/delete/` has a trailing slash but LegacyUrlConverter test works just fine

Comment (PierreRambaud) [reply src/PrestaShopBundle/Resources/config/routing/admin/improve/design/theme.yml]: it does, `/` at the end means directory.
If you try to go to http://ps-develop.localhost/admin-dev/index.php/sell/catalog/products/
it'll not work :D

Comment (matks) [reply src/PrestaShopBundle/Resources/config/routing/admin/improve/design/theme.yml]: So we should modify `/{themeName}/delete/` into `/{themeName}/delete` as well ?

Comment (PierreRambaud) [reply src/PrestaShopBundle/Resources/config/routing/admin/improve/design/theme.yml]: Yep!
Never end with a slash if it's not a directory :)

File: src/PrestaShopBundle/Resources/config/routing/admin/improve/design/theme.yml (hunk 1 above)
Comment (tomas862) [reply src/PrestaShopBundle/Resources/config/routing/admin/improve/design/theme.yml:15]: yes it truly is :D 
Screenshot from 2019-06-05 10-25-40

**Comment by tomas862** [comment]
now travis is green :tada:

**Review by matks — APPROVED** [review]

**Pull request merged**